samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/FAQ.md

```diff
@@ -0,0 +1,104 @@
+
+2. How to deploy FHIR loader?
+   + Go to [FHIR Bulk Loader](https://github.com/microsoft/fhir-loader) and follow the [deployment steps](https://github.com/microsoft/fhir-loader#deployment).
+
+3. How to configure Azure API for FHIR server for export?
+   + Follow steps [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/azure-api-for-fhir/configure-export-data) to configure settings for export on Azure API for FHIR server.
+
+4. Will there be any message to the user when export is finished?
+   + No, users must check the status manually using the get request in Postman with URL returned in the Content-Location response headers of export operation response.
+     
+     If the URL returns status 202 Accepted, it means the export job is still in progress, if the status is 200 Completed, it means the export job is completed.
+     
+     More details could be found in the readme document of the app [here](https://github.com/Azure-Samples/azure-health-data-services-samples/tree/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir).
+
+5. How do you check when the FHIR loader is done uploading?
+   + Users must check the containers i.e., "bundles" and "ndjson� to know about FHIR loader completion. FHIR Loader has completed loading when these containers are empty.
+
+6.	Do users need to wait for one export to finish, then do the import, wait for import to finish, then kick off the next export?
+    + Within each migration, users need to wait for export to finish before they can start import so FHIR Loader can import everything that was exported (i.e., wait for export#1 to finish before starting import#1). However, users can start export#2 while import#1 is finishing up. Users can only overlap though by a factor of 1, and exports need to be going to different containers.
+
+      Users should use _since and _till parameters with export.
+
+      On initial export from the source FHIR server, they should include "_till" in $export
+
+      After that initial migration, they should then run subsequent $export using "_since" and "_till".
+
+7.	Are there any configurations to be done for Destination FHIR server and how to do them?
+ 	+ For Destination FHIR server we need to configure number of RU, auto scaling, number of nodes and throttling limits, Configuration details could be found in document [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Server_%26_App_Config.md#destination-fhir-server).
+
+8.	What configurations to be considered for FHIR Bulk Loader Applications?
+	+ Users can configure App Service plan, number of instances, number of fhir resources per bundle, more details could be found [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Server_%26_App_Config.md#fhir-bulk-loader-application).
```

Comment:
We should also provide some guidance on the configuration of the FHIR Loader app layer. 

This doc has some information and Steve has some as well:
https://github.com/microsoft/fhir-loader/blob/main/docs/performance.md

Reply:
changes done as per review comments.